--- obexftp-0.23/configure.in~	2009-02-17 19:27:48.000000000 +0100
+++ obexftp-0.23/configure.in	2013-06-11 14:04:59.540638730 +0200
@@ -3,7 +3,7 @@
 AC_INIT(ObexFTP, 0.23, zany@triq.net)
 #AC_CONFIG_SRCDIR(src/foo.c)
 AM_INIT_AUTOMAKE(dist-bzip2)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 # note: AC_HELP_STRING is deprecated in autoconf 2.59 and later but
 # AS_HELP_STRING is not available in autoconf 2.57 or earlier.
