--- apps/Makefile.am	2006-03-26 00:11:59.000000000 +0100
+++ apps/Makefile.am-new	2007-05-28 13:27:40.657210186 +0200
@@ -20,7 +20,6 @@
 				bt_discovery.c
 
 obexftpd_SOURCES =		obexftpd.c obexftp_sdp.c obexftp_sdp.h
-stress_SOURCES =		stress.c
 discovery_SOURCES =		discovery.c
 
 bin_PROGRAMS =			obexftp obexftpd
--- configure.in	2006-05-16 16:50:29.000000000 +0200
+++ configure.in-new	2007-05-28 13:41:21.684223141 +0200
@@ -24,7 +24,6 @@
 # Checks for libraries.
 PKG_CHECK_MODULES(OPENOBEX,openobex)
 
-AM_ICONV
 dnl IRDA_CHECK
 BLUETOOTH_CHECK
 SDPLIB_CHECK
