diff -urNp -x '*.orig' obexftp-0.23.org/apps/obexftpd.c obexftp-0.23/apps/obexftpd.c
--- obexftp-0.23.org/apps/obexftpd.c	2009-02-17 19:27:48.000000000 +0100
+++ obexftp-0.23/apps/obexftpd.c	2021-05-07 23:03:16.308834002 +0200
@@ -901,7 +901,7 @@ reset:
 	}
 	printf("Waiting for connection...\n");
 
-	(void) OBEX_ServerAccept(handle, obex_event, NULL);
+	//(void) OBEX_ServerAccept(handle, obex_event, NULL);
 
 	while (!finished) {
 		//printf("Handling connection...\n");
