--- obexftp-0.20/multicobex/Makefile.am.orig	2005-11-15 23:18:15.000000000 +0100
+++ obexftp-0.20/multicobex/Makefile.am	2006-09-10 12:14:30.675653500 +0200
@@ -4,7 +4,7 @@
 				-I$(top_srcdir) \
 				-I$(top_srcdir)/includes
 
-LDADD =				@OPENOBEX_LIBS@ \
+libmulticobex_la_LIBADD =		@OPENOBEX_LIBS@ \
 				../bfb/libbfb.la
 
 multicobexincludedir =		$(includedir)/multicobex
--- obexftp-0.20/obexftp/Makefile.am.orig	2005-12-10 02:39:41.000000000 +0100
+++ obexftp-0.20/obexftp/Makefile.am	2006-09-10 12:16:03.561458500 +0200
@@ -4,11 +4,10 @@
 				-I$(top_srcdir) \
 				-I$(top_srcdir)/includes
 
-LDADD =                         @OPENOBEX_LIBS@
-#				@BLUETOOTH_LIBS@ \
-#				@USB_LIBS@ \
-#				../multicobex/libmulticobex.la \
-#				../bfb/libbfb.la
+libobexftp_la_LIBADD =		@OPENOBEX_LIBS@ \
+				@BLUETOOTH_LIBS@ \
+				../multicobex/libmulticobex.la \
+				../bfb/libbfb.la
 
 obexftpincludedir =		$(includedir)/obexftp
 
--- obexftp-0.20/swig/python/Makefile.am.orig	2006-03-26 00:12:24.000000000 +0100
+++ obexftp-0.20/swig/python/Makefile.am	2006-09-10 12:33:53.344315750 +0200
@@ -2,7 +2,7 @@
 pkgpyexec_LTLIBRARIES =	_obexftp.la
 _obexftp_la_SOURCES =	python_wrap.c
 _obexftp_la_CPPFLAGS =	$(SWIG_PYTHON_CPPFLAGS) -I$(top_srcdir)/obexftp
-_obexftp_la_LDFLAGS =	-module
+_obexftp_la_LDFLAGS =	-module -avoid-version
 _obexftp_la_LIBADD =	$(top_builddir)/obexftp/libobexftp.la \
 			$(top_builddir)/multicobex/libmulticobex.la \
 			$(top_builddir)/bfb/libbfb.la \
