--- obexftp-0.23/swig/ruby/ruby_wrap.c~	2009-02-17 19:30:40.000000000 +0100
+++ obexftp-0.23/swig/ruby/ruby_wrap.c	2010-03-02 14:42:43.359124637 +0100
@@ -2580,7 +2580,7 @@
     
     /* VALUE str = StringValue(argv[0]); // perhaps better? */
     arg2 = STR2CSTR(argv[0]);
-    arg3 = (int) RSTRING(argv[0])->len;
+    arg3 = (int) RSTRING_LEN(argv[0]);
     
     
     
--- obexftp-0.23/swig/ruby/ruby_wrap.c.orig	2025-07-30 00:53:47.459228395 +0200
+++ obexftp-0.23/swig/ruby/ruby_wrap.c	2025-07-30 01:10:12.116355514 +0200
@@ -2106,7 +2106,7 @@
   arg1 = (obexftp_client_t *)(argp1);
   {
     arg2 = proxy_info_cb;
-    arg3 = argv[0];
+    arg3 = (void*) argv[0];
   }
   obexftp_client_t_callback(arg1,arg2,arg3);
   return Qnil;
@@ -2579,7 +2579,11 @@
     
     
     /* VALUE str = StringValue(argv[0]); // perhaps better? */
+    #if defined(StringValuePtr)
+    arg2 = StringValuePtr(argv[0]);
+    #else
     arg2 = STR2CSTR(argv[0]);
+    #endif
     arg3 = (int) RSTRING_LEN(argv[0]);
     
     
